lib/membrane_rtc_engine/endpoints/webrtc/track_sender.ex

```diff
@@ -68,7 +68,7 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.TrackSender do
     Membrane.RTC.Utils.telemetry_register(telemetry_label)
 
     {actions, state} =
-      if playback_state == :playing do
+      if playback_state == :playing and Track.is_simulcast?(state.track) do
```

Comment:
It seems to me that we're not using the `VariantTracker` at all, which is good, but I think we're still adding it to `state.trackers` and it might be better to not set it to give us some sort of protection against regression - we'll be getting nil errors at the very least if VariantTracker is used for a non-simulcast track.

Reply:
That was the initial idea but I would have to add more check `Track.is_simulcast?` (https://github.com/jellyfish-dev/membrane_rtc_engine/blob/master/lib/membrane_rtc_engine/endpoints/webrtc/track_sender.ex#L282 and https://github.com/jellyfish-dev/membrane_rtc_engine/blob/master/lib/membrane_rtc_engine/endpoints/webrtc/track_sender.ex#L200) which would result in less redable code.

Alternatively we could always start variantTracker timer but create variant tracker with different `required_samples` param i.e.


```elixir
    tracker =
      if Track.is_simulcast?(state.track) do
        VariantTracker.new(variant)
      else
        # assume that non-simulcast tracks are always active
        VariantTracker.new(variant, 0)
      end
```